tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/test_lockscreen_unlock_to_camera_with_passcode.py

```diff
@@ -17,7 +17,8 @@ def setUp(self):
         # Turn off geolocation prompt
         self.apps.set_permission('Camera', 'geolocation', 'deny')
 
-        self.data_layer.set_setting('lockscreen.passcode-lock.code', self._input_passcode)
+        self.setPasscodeTo1337()
```

Comment:
I would indeed prefer implementing `setPasscodeTo` as well, but that would requires us to re-implement the [hashing scheme](https://en.wikipedia.org/wiki/Pbkdf2) in python. I am not sure we would win much from that.

Alternatively, we could pre-compute a few more values than just for `1337`, but so far I have not seen a test that actually changes the passcode from one value to another.